packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -169,46 +173,237 @@ export const ShopperBasketsMutations = {
     AddTaxesForBasket: 'addTaxesForBasket'
 } as const
 
-type ShopperBasketMutationType = typeof ShopperBasketsMutations[keyof typeof ShopperBasketsMutations]
+export type ShopperBasketMutationType = typeof ShopperBasketsMutations[keyof typeof ShopperBasketsMutations]
+
+/**
+ * @private
+ */
+export const getCacheUpdateMatrix = (customerId: string | null) => {
+    const updateBasketQuery = (basketId?: string) => {
+        // TODO: we're missing headers, rawResponse -> not only {basketId}
+        const arg = {basketId}
+        return basketId
+            ? {
+                  update: [queryMap.basket({basketId, arg})]
+              }
+            : {}
+    }
+
+    const removeBasketQuery = (basketId?: string) => {
+        const arg = {basketId}
+        return basketId
+            ? {
+                  remove: [queryMap.basket({basketId, arg})]
+              }
+            : {}
+    }
+
+    const invalidateCustomerBasketsQuery = (customerId: string | null) => {
+        // TODO: should we use arg here or not? The invalidate method does not need exact query key.
```

Comment:
We don't need `arg` for invalidations.
React Query `invalidateQueries` does a partial matching: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/b9b79787f11926e9e46b3c13bca8099a422b80dd/packages/commerce-sdk-react/src/hooks/utils.ts#L63
https://react-query-v3.tanstack.com/guides/query-invalidation#:~:text=and%20others%20that%20support%20partial%20query%20matching